releasenotes/notes/profile-upload-2.4-8277b229fda6ff53.yaml

```
@@ -0,0 +1,3 @@
+other:
+  - |
+    Profiler: upload profiles using new backend request format
```

```suggestion
    profiling: upgrades the profiler to support the ``v2.4`` backend API for profile uploads, using a new request format.
```

This suggestion is to match our current release note style guidelines. Feel free to change if I missed anything else that was important.